Makefile

```diff
@@ -0,0 +1,16 @@
+
+build:
+	go build -o terraform-provider-bluechi
```

Comment:
I am used to the `build` naming (since its purpose is to build), but using the executable name itself is also fine with me. 
Regarding the `SOURCES` list I am not sure. This is a small project with only flavor, so a simple `go build` seems fine. In podman it seems to be used to specify a superset and exclude certain files later. Is there a specific reason to explicitly list the files?

Reply:
I agree that `go build` is enough. The reason you need to populate `SOURCES` is for the dependency of the executable target.
If you still want to have a `build` target, you can still have it and make it an empty one that depends on `terraform-provider-bluechi`

Reply:
Added both, the `build` target (delegating to the `terraform-provider-bluechi`) and the `SOURCES` dependency.